sherpa/astro/ui/tests/test_astro_ui.py

```diff
@@ -29,7 +29,7 @@
 
 class test_ui(SherpaTestCase):
```

Comment:
@requires_data can also be promoted. There is in principle one test that might run without test data, but it's in a class with a setUp method that requires test data, so that method would not be run anyway (and maybe that's a problem)